js/pulse.js

```diff
@@ -50,10 +57,14 @@
              */
             this.initialize = function(adManagerController, playerId) {
                 amc = adManagerController; // the AMC is how the code interacts with the player
+                //Enable overlay close button right away for Pulse
+                amc.adManagerSettings["showNonLinearCloseButton"] = true;
```

Comment:
@eshaglund Can you address this one line?  We are going to be releasing our ooyala hosted solution for the player shortly, and this will override everyone's settings even if they aren't using pulse.